app/serializers/all_chatroom_serializer.rb

```diff
@@ -0,0 +1,23 @@
+class AllChatroomSerializer < ActiveModel::Serializer
+  attributes :id, :user
+  attribute :last_message, if: :messages_exist?
+
+  def user
+    {
+      name: self.object.users.where.not(id: current_user.id).first.name
```

Comment:
You could implement a method partner(current_user) on the chatroom model to make this call a little bit easier to read